notebooks/MAST/TESS/interm_tesscut_astroquery/interm_tesscut_astroquery.ipynb

```diff
@@ -0,0 +1,507 @@
+    "\n",
+    "This tutorial shows the users how to do the following: use astroquery.catalogs to query the TIC, use astroquery Tesscut to determine the number of sectors that contain our target and download a FFI cutout.\n",
+    "\n",
+    "\n",
+    "### Table of Contents \n",
+    "  [Astroquery Search of the TIC](#catalog_ID) <br> \n",
+    "  [Tesscut to Perform FFI Cutout](#tesscut_ID) <br>\n",
+    "  [Create Light Curve](#lightcurve) <br>\n",
+    "  [Additional Resources](#resources_ID) <br> \n",
+    "  [About this Notebook](#about_ID) "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "***"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Import Statements\n",
+    "<a id=\"imports_ID\"></a>\n",
+    "\n",
+    "We start with a few import statements.\n",
+    "- *numpy* to handle array functions\n",
+    "- *astropy.io fits* for accessing FITS files\n",
+    "- *astropy.wcs WCS* to interpret the World Coordinate Systems\n",
+    "- *matplotlib.pyplot* for plotting the data\n",
+    "- *astroquery.mast* for the catalogs and for tesscut.\n",
+    "\n",
+    "For ease of use later on, we also set the root url for the Tesscut package here.\n",
+    "\n",
+    "If your version of astroquery doesn't have tesscut, try\n",
+    "pip install astroquery==0.3.9.dev5086"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "from astroquery.mast import Catalogs\n",
+    "from astroquery.mast import Tesscut\n",
+    "from astropy.coordinates import SkyCoord\n",
+    "from astropy.wcs import WCS\n",
+    "from astropy.io import fits\n",
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "%matplotlib inline\n",
+    "\n",
+    "###CHANGE THE NEXT LINE in Pull Request ###\n",
+    "#This is the url we are using.\n",
+    "Tesscut._TESSCUT_URL = 'https://mastdev.stsci.edu/tesscut/api/v0.1/'"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# DELETE THIS CELL\n",
+    "\n",
+    "#Next line puts your mast api token in a useful variable for later.\n",
+    "#See https://auth.mast.stsci.edu/info\n",
+    "masttoken = %env MAST_DV_TOKEN\n",
+    "Tesscut._session.headers[\"Authorization\"] = \"token %s\" % masttoken"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "***"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a id=\"catalog_ID\"></a>\n",
+    "## Get Coordinates of the Target Using Astroquery Catalogs\n",
+    "\n",
+    "Here we use the astroquery.mast Catalogs query_criteria function to request the RA and Dec of the object with TIC ID = 261105201.   \n",
```

Comment:
I think this line should be deleted? Or are you planning to switch back to query criteria once the fix gets into Astroquery?